sensing/pointcloud_preprocessor/src/blockage_diag/blockage_diag_nodelet.cpp

```diff
@@ -106,7 +106,7 @@ void BlockageDiagComponent::filter(
   const PointCloud2ConstPtr & input, [[maybe_unused]] const IndicesPtr & indices,
   PointCloud2 & output)
 {
-  boost::mutex::scoped_lock lock(mutex_);
+  std::scoped_lock lock(mutex_);
```

Comment:
Don't you have to fix the type of `mutex` as well?
https://github.com/RyuYamamoto/autoware.universe/blob/f7a6f87db80d5b59b2fc7dcc6f6a25ce090043ac/sensing/pointcloud_preprocessor/include/pointcloud_preprocessor/filter.hpp#L170

Reply:
sorry, I forgot to fix boost::mutex. I will fix it.